modules/policy-shared-responsibility.adoc

```diff
@@ -9,8 +9,9 @@
 
 The customer and Red Hat share responsibility for the monitoring and maintenance of an {product-title} cluster. This documentation illustrates the delineation of responsibilities by area and task.
 
+[id="policy-shared-responsibility-incident-operations-management_{context}"]
```

Comment:
Should this be `incident-operations-management_{context}`. Same for the other id in this file.

Reply:
@vikram-redhat I had to add in a lot of missing IDs for the assemblies, and some of these titles alone could potentially conflict. To be safe, I started all of them with the main assembly's ID + the heading. Unless I'm missing something, it's not really that important what we call these IDs (unless you object to the length of the name).

Reply:
I was asking about the anchor ids, and not the actual content in these docs. I am sure @bergerhoffer added those ids :) 

I believe @bergerhoffer is reducing them to remove the file name.

Reply:
@vikram-redhat I went through and tried to reduce the size of the long ones I came across. I'm not sure why I forgot about the context making the IDs unique... I must have been delirious at that point :). If you see any other long ones that I missed, let me know and I'll update.